workflows/cloudify_system_workflows/snapshot.py

```diff
@@ -385,12 +385,86 @@ def _update_es_node(es_node):
             source['main_file_name'] = ''
 
 
+def _assert_clean_elasticsearch(es, log_warning=False):
+    """
+    Check if manager ElasticSearch is clean and raise error (or just
+    log warning) if it isn't.
+
+    :param es: ElasticSearch proxy object
+    :param log_warning: instead raising error just log warning
+    """
+
+    def _raise_or_log():
+        message = ('Manager is not clean, snapshot is allowed to be '
```

Comment:
The message should be different for both cases:
1. force=true: "Forcing snapshot restoration on dirty manager"
2. default: "Snapshot restoration on on a dirty manager is not permitted"

Feel free to adjust the messages.